### PR TITLE
Improved homepage (and meta descriptions)

### PR DESCRIPTION
**Pull request by j9t**
Closes DX-111

### CODE REVIEW

**Review by itay1313 — APPROVED** [review]
👌

File: pages/index.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ redirect_from:
           stroke="var(--body-text)" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
       </svg>
       <h3 class="welcome-title h3">Explore Functions</h3>
-      <p class="welcome-content">Learn how to create custom logic, flows and integrations for your business</p>
+      <p class="welcome-content">Learn how to create custom logic, flows, and integrations for your business</p>
```
Comment (lkisoslive) [inline pages/index.md:55]: @j9t I don't think this comma is needed here and prefer we stick to the current method of no comma in a list of 3+ before and.

File: pages/index.md [hunk 2]
```diff
@@ -117,8 +117,8 @@ redirect_from:
         <path d="M31.1667 24.1693L23.8427 22.2507" stroke="var(--body-text)" stroke-width="2" stroke-linecap="round"
           stroke-linejoin="round" />
       </svg>
-      <h3 class="welcome-title h3">Get Started</h3>
-      <p class="welcome-content">Get up and running quickly using our getting started guides, APIs, and SDKs</p>
+      <h3 class="welcome-title h3">Get started</h3>
+      <p class="welcome-content">Get up and running quickly using our “Getting Started” guides, APIs, and SDKs</p>
```
Comment (lkisoslive) [inline pages/index.md:121]: @j9t again no comma needed before and

File: pages/index.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ redirect_from:
 ---
 
 <div class="header-label">
-  We empower developers to build innovative solutions that are tailored to their business needs to best manage and automate all consumer conversations.
+  We empower developers to build innovative solutions to manage, customize, and automate consumer conversations tailored to their business needs.
```
Comment (lkisoslive) [inline pages/index.md:11]: @j9t no comma needed before and

File: pages/index.md [hunk 4]
```diff
@@ -152,15 +151,15 @@ redirect_from:
         <path d="M6.5 6H20.5V14H6.5V6Z" stroke="var(--body-text)" stroke-width="2" stroke-linecap="round"
           stroke-linejoin="round" />
       </svg>
-      <h3 class="welcome-title h3">Stay Informed</h3>
-      <p class="welcome-content">Get LivePerson’s latest news, updates and releases</p>
+      <h3 class="welcome-title h3">Stay informed</h3>
+      <p class="welcome-content">Get LivePerson’s latest news, updates, and releases</p>
```
Comment (lkisoslive) [inline pages/index.md:155]: no need for a comma before and

**Review by lkisoslive — COMMENTED** [review]
Good to go but provided feedback regarding grammar, otherwise all set.

**Comment by j9t** [comment]
@lkisoslive, we had decided on using the Oxford comma (see [our style guide](https://github.com/LivePersonInc/developer-center#notes-on-content-and-code)), and off the top of my head it’s even in the brand guidelines. Was there anything else that caught your eye?

**Comment by lkisoslive** [comment]
No, I see you implemented my feedback regarding the header-label and it looks good. Thanks! @j9t

**Comment by j9t** [comment]
Thanks @lkisoslive! Do you mind approving?

**Review by lkisoslive — APPROVED** [review]
Looks great!

**Comment by j9t** [comment]
Thanks @lkisoslive!

**Pull request merged**